### ISSUE TITLE
Add Repo Name to Each Extension

### ISSUE THREAD

**Issue by jeyemrs** [aniyomiorg/aniyomi#1622]
### Describe your suggested feature

Since official extensions are gone, users will be depending on repos. I myself have more than one repo, and it's such a hassle to distinguish where the extension came from. So for a suggestion, in the settings, whenever you're going to add a repo, there should be a field where you can add a name for the repo, like this:

Name:
Repo URL:
 
After that, the name will show beside the version number of each extension, like how OBSOLETE works. Adding a name to the repo may be optional if the user doesn't want to.

This will be useful for the user to figure out right away where to report a broken extension.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.3.0](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by hollowshiroyuki** [comment]
this may be implemented once mihonapp/mihon#506 is merged ig

**Comment by Iffuu01** [comment]
Repo for aniyomi
